schemas/database/015_add_constraints_for_identities.down.sql

```diff
@@ -0,0 +1,7 @@
+DROP UNIQUE INDEX idx_unique_insertion_leaf;
```

Comment:
This down migration doesn't work

First I'm getting
```
error: while executing migrations: error returned from database: syntax error at or near "UNIQUE"
```

then after removing both `UNIQUE` keywords on the first 2 lines I get:
```
error: while executing migrations: error returned from database: syntax error at or near ";"
```

Reply:
Yeah...fixed. :)